src/test/java/com/google/api/codegen/testsrc/common/tagger.proto

```diff
@@ -38,7 +38,7 @@ message AddTagRequest {
   // projects/{project}/zones/{zone}/disks/{disk}.
   string resource = 1 [
     (google.api.field_behavior) = REQUIRED,
-    (google.api.resource_reference) = "google.example.library.v1.Book"];
+    (google.api.resource_reference).type = "library.googleapis.com/Book"];
```

Comment:
Do all of these resource_reference.types have to be fully qualified like this?

Reply:
As I understand it, yes.